source/Calamari/Kubernetes/Scripts/AzurePowershellContext.ps1

```diff
@@ -11,6 +11,9 @@ $OctopusAzureEnvironment = $OctopusParameters["Octopus.Action.Azure.Environment"
 $OctopusOpenIdJwt = $OctopusParameters["Octopus.OpenIdConnect.Jwt"]
 $OctopusUseOidc = ![string]::IsNullOrEmpty($OctopusOpenIdJwt)
 
+## The script is passed the following parameters.

```

Comment:
Which version of Server are you targeting? I've made a breaking change in Calamari (https://github.com/OctopusDeploy/Calamari/pull/1405) so you'll need to target a 28.x.x branch if this is going to **2024.4**

Reply:
I think it'll need to be 2025.1 - so main is fine (i.e. I don't really want to push this out to on-prem).
Aside from that - the feature toggle should probably be flipped to prevent the usage of AzureRM by default (but with an escape hatch).